src/app/pages/common/pages-home-page/pages-home-page.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container home-page">
+    <div>
+        <img src="assets/images/logos/home-page-logo.svg" alt="track-expenses-logo"/>
```

Comment:
Alt should be a description of the image, so something like "A logo of Track Expenses App"